spec/rollbar/encoding/encoder_spec.rb

```diff
@@ -0,0 +1,63 @@
+# encoding: utf-8
+require 'spec_helper'
+
+unless RUBY_VERSION.start_with?('1.8')
+  require 'rollbar/encoding/encoder' unless
```

Comment:
That `require` and `unless` is something weird. I'll review it.

@joneslee85 I understand your point, but I'd prefer to just test that class separately.